_posts/2020-03-31-DevOpsDays-2019.md

```diff
@@ -0,0 +1,236 @@
+
+Charity Majors is a co-founder and engineer at [Honeycomb.io](https://www.honeycomb.io){:target="_blank" rel="noopener noreferrer"}, a monitoring and analyzer tool that can gather information from various sources to provide an overview of what's happening in your (production) environment. 
+
+Most of you have experience with sieving information through logs and analyzing all kinds of dashboards to find and fully understand what went wrong in your application. 
+Finding and understanding the data can be really hard. 
+You lack the bigger picture because the tools for doing metrics, logging and tracing (what she calls the “three pillars of observability”) have their own benefits and drawbacks. 
+
+Logs can be useful for compliance reasons but they are not so great to analyze the context. 
+Metric tools are maybe more visual than logs, but they are less ideal for debugging purposes to completely understand why something failed. 
+
+We as human beings like visual representation to easily perceive why certain situations could occur. 
+For example during and after deploying new versions because the chance is higher that something will go wrong. 
+According to Charity, deployments are not like 'flipping a binary switch'. 
+It's more like a step in a long process of being confident in your code that it will work on other environments rather than just on your local machine or development environment.
+
+Especially deployments to production will not always go as planned and it will be even more difficult to do so, since infrastructure has become more complex over the recent years. 
+In comparison with a decade ago where most infrastructure was just as simple as a basic LAMP stack that hosts some PHP applications. 
+Nowadays you have replica sets, nodes and your app that’s built upon a micro-service.
+
+It would be very difficult to quickly grasp what’s going on with your applications and its underlying systems without good observability. 
+
+Observability is a measure of how well internal states of a system can be interred from knowledge of its external outputs.
+It has the advantages of logging, tracing and metrics, and could be achieved with the right tools, if set up correctly which is something that Honeycomb is trying to achieve. 
+
+It is also good to know that observability is not the same as monitoring. 
+That is because monitoring is more like a ‘post-talk’ approach in her eyes. 
+You’re typically too late and the damage has already been done upon discovery.
+Monitoring systems have not changed significantly in twenty years and have fallen behind the way we build software. 
+Software nowadays consists of large distributed systems that are made up of many non-uniform interacting components, while the core functionality of monitoring systems has stagnated.
+
+## How Convenience is Killing Open Standards, by Bernd Erk
+
+You may know that interchanging configurations and such between cloud providers can be challenging. 
```

Comment:
You may know that migrating configurations from one cloud provider to another can be challenging.